lib/proc-macros/src/content_hash.rs

```diff
@@ -10,7 +10,7 @@ pub fn generate_hash_impl(data: &Data) -> TokenStream {
                 let hash_statements = fields.named.iter().map(|f| {
                     let field_name = &f.ident;
                     quote_spanned! {f.span()=>
-                        crate::content_hash::ContentHash::hash(&self.#field_name, state);
+                        jj_lib::content_hash::ContentHash::hash(&self.#field_name, state);
```

Comment:
nit: maybe better to use `::jj_lib` in generated code?

Reply:
Thanks I agree. Should be fixed in the latest diff.